drsm/claim.go

```diff
@@ -11,9 +11,9 @@ import (
 )
 
 func (d *Drsm) podDownDetected() {
-	fmt.Println("started Pod Down goroutine")
+	logger.DrsmLog.Infoln("started Pod Down goroutine")
 	for p := range d.podDown {
-		logger.DrsmLog.Infoln("pod Down detected", p)
+		logger.DrsmLog.Infoln("pod Down detected %v ", p)
```

Comment:
This change will not work. Either, leave it as it was before, or change it as follows:
```suggestion
		logger.DrsmLog.Infof("pod Down detected %v", p)
```